templates/windows/chocolateyUninstall.ps1.erb

```diff
@@ -0,0 +1,20 @@
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+$contentFile=(join-path -path "$toolsDir" -childpath "file-list") + ".txt"
+if (Test-Path -path "$contentFile") {
+  Write-Debug "Removing files defiend in `'$contentFile`'"
+  $contentFile
+  $contents=get-content "$contentFile"
+  foreach ($fileInstalled in $contents) {
+    $fileInstalled=$fileInstalled.Replace("/cygdrive/c", "C:")
+    $fileInstalled=$fileInstalled.Replace("/", "\")
+    $fileInstalled
+    if ($fileInstalled.PSIsContainer) {
+      Write-Debug "Not removing directory $fileInstalled"
+    } else {
+      remove-item -Path "$fileInstalled" -ErrorAction SilentlyContinue -Force
+    }
+  }
+} else {
+  Write-Debug "'$contentFile' not found, not removing any files in non-standard chocolatey location"
+}
```

Comment:
Fixed now :) Uninstall succeeds, but leaves a lot of the directory structure on the file system. I did my best to thoroughly clean it up, but wasn't entirely successful